README.md

```diff
@@ -307,6 +309,51 @@ python3 measure.py
 
 See the [list](docs/supported_models.md) of supported lights which don't need any manual configuration
 
+## Setting up for energy dashboard
+If you want to use this power sensors with the new [energy integration](https://www.home-assistant.io/blog/2021/08/04/home-energy-management/), you have to create a energy sensor which utilizes the power of the powercalc sensor. This can be done with the [Riemann integration integration](https://www.home-assistant.io/integrations/integration/), which calculates the energy and since release 2021.8 this integration can natively be used in the energy dashboard. An example configuration, which you have to copy into your `configuration.yaml`, can be seen below:
+
```

Comment:
We assume you have a powercalc sensor with entity_id `sensor.kingkong_power`

Reply:
Didn't change it word by word, but meaning should be right

Reply:
✔️